config/unichain-sepolia.template.json

```diff
@@ -0,0 +1,8 @@
+{
+  "network": "unichain-testnet",
+  "address": "0x22C1f6050E56d2876009903609a2cC3fEf83B415",
+  "startBlock": 13046450,
+  "file": "XPoap",
+  "tokenEvent": "EventToken(indexed uint256,uint256)",
+  "description": "<ADD ME>"
```

Comment:
```suggestion
  "description": ""
```

Or something?

Reply:
We have been using "<ADD ME>" in all of the templates

Reply:
Why? Leaving TODO later in life?

Reply:
It's a template and what it has to change is the description. So we don't collide with other subgraphs deployed in  goldsky.

If we hardcode it here then anyone can deploy it and sort of break our subgraph

Reply:
Thanks for the explanation.